src/services/Authentication/core/Common/Logging/LoggerManager.cs

```diff
@@ -0,0 +1,13 @@
+using Serilog;
+
+namespace Common.Logging;
+
+public class LoggerManager : ILoggerManager
+{
+    private static readonly ILogger logger = Log.Logger;
+
```

Comment:
it is better to use DI and pass ILogger through the constructor, rather than create it statically